test/functional/pdo_sqlsrv/PDO100_InsertNulls.phpt

```diff
@@ -82,32 +90,19 @@ SQL
 // Repro
 //
 //--------------------------------------------------------------------
-function Repro()
-{
-    $failed = null;
-    
-    $testName = "PDO - Insert Nulls";
-
-    StartTest($testName);
-
-    try
-    {
-        $failed |= InsertNullsTest(PDO::PARAM_LOB);
-        $failed |= InsertNullsTest(PDO::PARAM_STR);
-    }
-    catch (Exception $e)
-    {
-        echo $e->getMessage();
-    }
-
-    if ($failed)
-        FatalError("Possible Regression: Could not insert NULL");
-    
-    EndTest($testName);
+try {
+    $failed = false;
+    $failed |= insertNullsTest(PDO::PARAM_LOB);
```

Comment:
why changed _failed_ from null to false?

Reply:
I just think it makes more sense to or a boolean